docs/_includes/default.html

```diff
@@ -37,14 +36,14 @@ <h1>
     <main id="content">{{ content }}</main>
 
     <footer>
-      <span>
+      <div>
         <a href="https://mochajs.org">mochajs.org</a> is licensed under a
-        <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"
+        <a rel="license" external href="http://creativecommons.org/licenses/by/4.0/"
           >Creative Commons Attribution 4.0 International License</a
         >.
         <p>
           <em>Last updated: {{ 'now' | date: '%a %b %d %H:%M:%S %Y' }}</em>
-        </p></span
+        </p></div
```

Comment:
Put `<div>` on its own line

Reply:
This shouldn't be using `<em>` unless it's ***actually emphasis*** (e.g., like that right there).

I recommend either a plain `<i>` or simply targeting it with a structural selector.